kuzzleio/kuzzle
# Register subscription

## What does this PR do ?
Closes  #1667 by introducing the `registerSubscription`and `unregisterSubscription` methods in the Plugin Context.

### How should this be manually tested?
Write a plugin that exposes a custom action. In this action, use `registerSubscription` to subscribe to a given collection with a given set of filters. Then, connect to the backend via `wscat` and trigger the custom action. From now on, the `wscat` session should receive notifications for the subscription defined by the call to `registerSubscription`.

### TODO
- [ ] `unregisterSubscription` method
- [ ] put the methods into a more semantic namespace in the Plugin Context (like `realtime`)

### Other changes

--

### Boyscout

--
